src/pages/driver/share.tsx

```diff
@@ -0,0 +1,87 @@
+import NestedLayout from "@/modules/layouts/Nested";
+import { Box, Button, Flex, Input, Textarea } from "@chakra-ui/react";
+import { PropsWithChildren } from "react";
+
+const FormGroupTitle = ({ children }: PropsWithChildren) => (
+  <Box fontSize="18px" fontWeight="500">
+    {children}
+  </Box>
+);
+
+const TripPointsForm = ({ form }: { form: string }) => (
+  <Flex gap="16px" direction="column">
+    <FormGroupTitle>Your Trip</FormGroupTitle>
+    <Input form={form} name="starting-point" placeholder="Starting point" />
+    <Input form={form} name="destination" placeholder="Destination" />
+  </Flex>
+);
+
+const current = new Date();
```

Comment:
This should be placed in the component that uses it.
`const current` now has the value of the time when nextjs compiles it (when running`next build` during deployment (CI/CD workflow))